tests/unit/router/helpers/PrivateRoute.spec.jsx

```diff
@@ -0,0 +1,53 @@
+import { renderWithProviders } from '~tests/test-utils'
+import { Navigate, Outlet, useOutletContext } from 'react-router-dom'
+import PrivateRoute from '~/router/helpers/PrivateRoute'
+
+vi.mock('react-router-dom', async () => {
+  const actual = await vi.importActual('react-router-dom')
+  return {
+    ...actual,
+    Navigate: vi.fn(() => null),
+    Outlet: vi.fn(() => null),
```

Comment:
I guess, you can leave just vi.fn() and it will work the same

Reply:
there will be an error
<img width="941" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/79856961/b3396a82-0309-494a-a1fd-28ad819c9f02">